index.html

```diff
@@ -12,7 +12,6 @@
         <div class="wrapper" id="wrapper">
             <header class="header">
```

Comment:
- [ ] Please remove the style-agent margin on the `body` element. You can do that by 

```
body {
  margin: 0
}
```